build.gradle

```diff
@@ -32,11 +32,16 @@ dependencies {
 }
 
 application {
-  // Define the main class for the application
+  // Define the main class for the application.
   mainClass.set('hellos.Hellos')
 }
 
 test {
-  // Use junit platform for unit tests
+  // Use junit platform for unit tests.
   useJUnitPlatform()
 }
+
+tasks.withType(JavaCompile) {
```

Comment:
Have we confirmed that this applies to the test code too?

Reply:
I checked—you were correct, it applies to both app code and test code! :)